ingest-app/docs/api.md

```diff
@@ -481,3 +481,145 @@ Example output:
+
+Bulk update is initiated through an ingest POST endpoint with the concept ids to update, the update type, the update field, and update information. The metadata is converted to the latest version of UMM, if not the native format, updated according to the parameters, and saved as the latest version of UMM, **making the native format of the collection now UMM**. Previous revisions of the collection are retained in the original native format.
+
+Updated collections are validated through ingest validation and the updates will not be saved if the updated collection is not valid UMM or fails business rule validation. The error will be recorded in the individual collection status, which can be queried via the status endpoint. Collection validation warnings will not prevent saving the updated collection and the warnings will be recorded in the individual collection status.
+
+Bulk update currently supports updating the following fields:
+
+  * Science Keywords
+  * Location Keywords
+  * Data Centers
+  * Instruments
+  * Platforms
+
+Instruments are nested within platforms so instrument updates are applied to all platforms in the collection.
+
+The following update types are supported:
+
+  * Add to existing - the update value is added to the existing list. An update value is required.
+  * Clear all and replace - clear the list and replace with the update value.
+  * Find and replace - replace any instance in the list that matches the find value with the update value.
+  * Find and remove - remove any instance from the list that matches the find value.
+
+Update types that include a find will match on the fields supplied. For example, for a science keyword update with a find value of {"Category": "EARTH SCIENCES"}, any science keyword with a category of "EARTH SCIENCES" will be considered a match regardless of the values of the science keyword topic, term, etc.
+
+When a value is replaced by bulk update, only the field(s) specified in the update value will be replaced with the rest of the original value retained. For example, with a platform update value of {"ShortName": "A340-600"}, only the short name will be updated during a find and replace, while the long name, instruments, and other fields retain their values.
+
+If multiple bulk updates are run at the same time with the same concept-ids, there is no guarantee of the order that the updates will be performed on a collection. For example, if a clear all and replace is initiated, then an add to existing on the same collection, the clear all and replace could happen after the add to existing. Because of this, it is best to not run bulk update operations in parallel on overlapping collections.
+
+### Initiate Bulk Update
+
+Bulk update can be initiated by sending an HTTP POST request to `%CMR-ENDPOINT%/providers/<provider-id>/bulk-update/collections`
```

Comment:
Quick question: you generated this locally and confirmed that `%CMR-ENDPOINT%` was what you expected (for a local run)? The reason I ask is that I want to make sure that the changes I made previously to the docs generation still support this ...